### PR TITLE
Parcelshop locator

### PR DESCRIPTION
**Pull request by timoj**
## Description
- Added parcelshop locator
- Added config for dispatch selection for locator
- Added config for en-/disabling the locator
- Added checkout validation for the locator

## References

Jira: [WWE-25](https://wuunder.atlassian.net/secure/RapidBoard.jspa?rapidView=6&projectKey=WWE&modal=detail&selectedIssue=WWE-25)
## Tasks

* [x] update CHANGELOG.md
* [x] reviewed (by @Mittows )
* [x] merged with origin/development

### CODE REVIEW

File: Wuunder/Views/backend/wuunder/controller/list.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ Ext.define('Shopware.apps.Wuunder.controller.List', {
             success: function (response, opts) {
                 var data = Ext.decode(response.responseText);
                 Ext.util.Cookies.set('wuunderOrderOverviewAfterRedirect', 1);
+                console.log(data);
```
Comment (Mittows) [inline Wuunder/Views/backend/wuunder/controller/list.js]: Remove loggin?

File: Wuunder/Views/backend/wuunder/view/list.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ Ext.define('Shopware.apps.Wuunder.view.List', {
             tooltip: 'Ship with Wuunder',
             dataIndex: 'wuunderShipmentData',
             getClass: function (value, meta, record, rowIndex, colIndex, store) {
+                console.log(record);
```
Comment (Mittows) [inline Wuunder/Views/backend/wuunder/view/list.js]: Remove logging?

File: Wuunder/Views/backend/wuunder/controller/list.js (hunk 1 above)
Comment (timoj) [reply Wuunder/Views/backend/wuunder/controller/list.js]: Removed

File: Wuunder/Views/backend/wuunder/view/list.js (hunk 2 above)
Comment (timoj) [reply Wuunder/Views/backend/wuunder/view/list.js]: Removed

File: Wuunder/vendor/nategood/httpful/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Httpful
```
Comment (Mittows) [inline Wuunder/vendor/nategood/httpful/README.md]: Should this file/folder be included?

Comment (timoj) [reply Wuunder/vendor/nategood/httpful/README.md]: removed, .gitignore was updated after commit

**Pull request merged**